frontend/src/index.css

```diff
@@ -161,7 +161,7 @@ code {
 
 .contentCenter {
   margin-top: 30px;
-  width: 900px;
+  width: 1000px;
```

Comment:
Assuming this line is not about the template? If you didn't mean to make this change can you revert it?

Reply:
Hmm, it's actually the container for the editorbox which fits the template in, so it effects how it looks.  Do you want me to change it in another pull request instead?

Reply:
Oh I understand. I was worried about breaking another component's style. Thank you, I'll merge this